uber/templates/registration/attendee_data.html

```diff
@@ -55,7 +55,7 @@
       }, 1000);
 };
 $(window).on("runJavaScript", function () {
-if($.field('return_to').length) {
+if($.field('return_to') && $.field('return_to').length) {
```

Comment:
I'm not sure this extra ``&&`` does anything.  ``$.field`` returns ``null`` if there's no match, and it returns a jQuery object if there's a match, so I don't think there's any situation where it would return a result which is truthy but with length 0.

Reply:
Hm... okay. I don't know how it works I guess, I thought you had to check for length to be sure a field is really there.

Reply:
That is true for general jQuery queries but not for ``$.field`` specifically.

So what you were probably thinking of is the fact that if you say

```javascript
if ($('.something')) { ... }
```

then that if-statement will always evaluate to true, because unlike in Python (where empty lists are falsey) empty collections in Javascript are truthy.

But the ``$.field`` method has differently semantics and returns ``null`` rather than an empty collection if the field isn't present, specifically to simply if-statements like this.

Reply:
Ohhhhhh

Reply:
OK, I've removed the code in question.